core/src/main/java/com/group5/core/world/DirPlayerTextures.java

```diff
@@ -0,0 +1,48 @@
+package com.group5.core.world;
+
+import java.util.ArrayList;
+
+import com.badlogic.gdx.graphics.Texture;
+import com.group5.core.EndlessRunner;
+
+/**
+ * In this class we store the textures for the player.
+ * @author Nils
+ */
+public class DirPlayerTextures {
+
+    /**
+     * List in which the player are stored.
+     */
+    private ArrayList<Texture> tlist;
+
+    /**
+     * Directory in which the textures for the player are stored.
+     */
+    public DirPlayerTextures() {
+        tlist = new ArrayList<Texture>();
+        addDefaultTextures();
+    }
+
+    /**
+     * This methods adds the different textures in an arraylist<Texture>.
+     */
+    private void addDefaultTextures() {
+        if (tlist.size() != 0) {
+            throw new Error("Something went wrong, mr/ms programmer...");
+        }
+        Texture t1 = EndlessRunner.get().getTextureCache().load("chickentimejumptest.png");
```

Comment:
I don't think it's nice that this class statically defines loading this exact pic. I think it would be better to have a general class that can get textures or the way to get them through the texture cache and assign keys to such a thing which an worldobject with switching states asks for.